docs/source/create_channel/create_channel_config.md

```diff
@@ -0,0 +1,296 @@
+
+### Channel modification policies
+
+The channel **structure** is governed by modification policies within the channel configuration. Every group within the channel configuration has a modification policy that needs to be satisfied in order to update the information contained within the group. As example, the group that defines an organization, the group the defines the complete set of channel members, and the group that defines the channel consenter set each have a different modification policy.
+
+Each modification policy can reference an ImplicitMeta policy or a signature policy. For example, the group that the defines each organization references the `Admins` signature policy associated with that organization. As a result, an organization can update their channel MSP or set an anchor peer without approval from other channel members. The modification policy of the application group that defines the set of channel members is the `Channel/Application/Admins` ImplicitMeta policy. As a result, the default policy is that a majority of organizations need to approve the addition or removal of a channel member.
+
+### Channel policies and Access Control Lists
+
+The policies within the channel configuration are also referenced by [Access Control Lists (ACLs)](../access_control.html) that are used to restrict access to Fabric resources used by the channel. The ACLs extend the policies within the channel configuration to govern the **processes** of the channel. You can see the default ACLs in the [sample configtx.yaml file](http://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml). Each ACL refers to a channel policy using the path. For example, the following ACL restricts who can invoke a chaincode based on the `/Channel/Application/Writers` policy:
+```
+# ACL policy for invoking chaincodes on peer
+peer/Propose: /Channel/Application/Writers
+```
+
+Most of the default ACLs point to the ImplicitMeta policies in the application section of the channel configuration. To extend the example above, an organization can invoke a chaincode if they if the can satisfy the `/Channel/Application/Writers` policy.
+
+  ![channel writer policies](application-writers.png)  
+
+*Figure 4: The peer/Propose ACL is satisfied by the /Channel/Application/Writers policy. This policy can be satisfied by a transaction submitted by a client application from any organization with the writers signature policy.*
```

Comment:
These diagrams are really helpful. One thing that occurred to me is that if we included examples of the signature policies in the right column, they would tie all this together.

So for example in this diagram, instead of in the Org1 box where you have Writers, we could have 
 Writers "OR('Org1MSP.admin', 'Org1MSP.client')"

Just an idea. I know that could be alot of work and might clutter up your diagrams, but for me it visually ties the ImplicitMeta and Signature policies together.

Reply:
I like the idea, but I think it would get awfully busy, so I will leave for now.